apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -221,6 +222,9 @@ export default class Simple2Sequencer extends Sequencer {
       length: soundData.length,
       soundType: soundData.type,
       blockId,
+      validationInfo: {
+        parentControlTypes: findParentStatementInputTypes(blockId),
+      },
       ...this.getCommonEventFields(),
```

Comment:
optional: could we pass `blockId` to `getCommonEventFields()` and assign `validationInfo` there if it's present on all events?

Reply:
Sure! Done. This method only exists for Simple2 so I left the Advanced Mode implementation as it was.